app/javascript/react/components/Graph/Graph.tsx

```diff
@@ -28,6 +32,17 @@ const Graph = (props: HighchartsReact.Props) => {
 
   const yAxisOption = getYAxisOption(thresholdsState);
 
+  const onMouseOverSingle = (latLng: {
+    latitude: number;
+    longitude: number;
+  }) => {
+    console.log(latLng);
+  };
+
+  const onMouseOverMultiple = (xLess: number, xMore: number) => {
+    console.log(xLess, xMore);
+  };
```

Comment:
Do we want it to stay?

Reply:
no, but i think this is from older commit